app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class TargetsController < ApiController
+      def create
+        current_user.targets << target
```

Comment:
What if this fails? Does it raise an exception?

Reply:
Yeah, the question is not where we are catching this, but if this is rasing an exception or not. If this doesn't raise an exception (acts as `save` instead of `save!` for instance) then we must change it to ensure that it is being raised.

Anyway, I think you already updated this code